resources/lib/syncEpisodes.py

```diff
@@ -208,9 +208,11 @@ def __traktLoadShows(self):
                 int(y), line2=kodiUtilities.getString(32102) % (i, x))
 
             # will keep the data in python structures - just like the KODI response
-            show = show.to_dict()
-
-            showsWatched['shows'].append(show)
+            show_dict = show.to_dict()
+            # reset_at is not included when calling `.to_dict()`
+            # but needed for watched shows to know whether to reset the watched state
+            show_dict['reset_at'] = utilities.to_iso8601_datetime(show.reset_at) if hasattr(show, 'reset_at') else None
```

Comment:
The `reset_at` is not needed on a show, and while `Show` extends `Media` in the trakt.py library, it is not included when calling `to_dict()` on a show.